crates/wysiwyg/src/tests/testutils_composer_model.rs

```diff
@@ -208,13 +205,36 @@ pub fn tx(model: &ComposerModel<u16>) -> String {
             }
         }
         Range::NoNode => panic!("No node!"),
-        Range::TooDifficultForMe => {
-            dbg!((model.state.start, model.state.end));
-            todo!("Range too difficult!")
+        Range::MultipleNodes(locations) => {
```

Comment:
This is awesome! Can we have some tests for it in `testutils_composer_model.rs` please?

Reply:
Added some new test cases to `cm_and_tx_roundtrip` with multiple nodes, fixed `tx` since it didn't apply the selection properly.